### PR TITLE
RELATED: RAIL-196 Added check for protected data in errorHandler

### PR DESCRIPTION
**Pull request by lhorak**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/496/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/399/ : FAILURE in 2m 43s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/506/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/411/ : FAILURE in 2m 09s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/518/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/423/ : FAILURE in 3m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/519/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/424/ : FAILURE in 1m 26s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/520/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/425/ : FAILURE in 2m 01s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/521/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/426/ : FAILURE in 1m 38s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/522/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/427/ : FAILURE in 1m 09s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/523/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/428/ : FAILURE in 1m 22s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/524/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/429/ : FAILURE in 1m 34s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/525/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/430/ : FAILURE in 1m 30s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/526/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/431/ : FAILURE in 1m 28s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/527/ : SUCCESS in 37s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/432/ : FAILURE in 3m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/528/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/433/ : FAILURE in 1m 56s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/529/ : SUCCESS in 41s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/434/ : SUCCESS in 2m 06s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/530/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/435/ : SUCCESS in 3m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/531/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/436/ : SUCCESS in 2m 45s

File: src/execution/fixtures/ExecuteAfm.fixtures.ts [hunk 1]
```diff
@@ -174,15 +174,45 @@ const tooLargeResponse: Execution.IError = {
     name: 'Error 413',
     message: 'Response is too large',
     response: {
-        status: 413
+        status: 413,
+        body: {} as ReadableStream, // tslint:disable-line
```
Comment (zbycz) [inline src/execution/fixtures/ExecuteAfm.fixtures.ts]: `body: null as ReadableStream | null,`
taken from `lib.es6.d.ts` :-)

`+` maybe add // TODO will be solved in RAIL-650

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/533/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/438/ : SUCCESS in 2m 45s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/537/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/442/ : FAILURE in 2m 40s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/538/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/444/ : SUCCESS in 2m 31s

File: src/execution/fixtures/ExecuteAfm.fixtures.ts [hunk 2]
```diff
@@ -174,15 +174,49 @@ const tooLargeResponse: Execution.IError = {
     name: 'Error 413',
     message: 'Response is too large',
     response: {
-        status: 413
+        status: 413,
+        body: null as ReadableStream | null, // tslint:disable-line
```
Comment (zbycz) [inline src/execution/fixtures/ExecuteAfm.fixtures.ts]: This should be valid TS, you may leave out the disable-line

File: src/helpers/tests/errorHandlers.spec.ts [hunk 3]
```diff
@@ -9,28 +9,82 @@ import {
...
-        error.response.status = 204;
-        expect(() => { convertErrors(error); }).toThrow(ErrorStates.NO_DATA);
+    return error;
+}
 
-        error.response.status = DataErrorCodes.HTTP_TOO_LARGE;
-        expect(() => { convertErrors(error); }).toThrow(ErrorStates.DATA_TOO_LARGE_TO_COMPUTE);
+describe('convertErrors', async () => {
+    it('should throw correct ErrorStates', async () => {
+        expect.assertions(7);
```
Comment (zbycz) [inline src/helpers/tests/errorHandlers.spec.ts]: 👍

**Comment by zbycz** [comment]
ok to test

**Comment by zbycz** [comment]
ok to test

**Comment by zbycz** [comment]
ok to test

**Comment by lhorak** [comment]
ok to test

**Comment by lhorak** [comment]
ok to test

**Pull request closed without merging**